cvFit/src/app/exercises/workoutList/dataTransfer.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from "@angular/core";
+
+@Injectable()
+export class TransferService {
```

Comment:
This is another one where I'm not sure what it's doing. This looks like a service to share data with other components? The variables and actions could be better with some renames. Then maybe when we understand what it's doing we might not even have a need for the service at all.